docs/topics/inheritance.md

```diff
@@ -0,0 +1,218 @@
+[//]: # (title: Inheritance)
+
+All classes in Kotlin have a common superclass `Any`, that is the default superclass for a class with no supertypes declared:
+
+```kotlin
+class Example // Implicitly inherits from Any
+```
+
+`Any` has three methods: `equals()`, `hashCode()` and `toString()`. Thus, they are defined for all Kotlin classes. 
+
+By default, Kotlin classes are final: they can’t be inherited.
+To make a class inheritable, mark it with the `open` keyword.
+
+```kotlin
+open class Base //Class is open for inheritance
+
+```
+
+To declare an explicit supertype, place the type after a colon in the class header:
+
+```kotlin
+open class Base(p: Int)
+
+class Derived(p: Int) : Base(p)
+```
+
+If the derived class has a primary constructor, the base class can (and must) be initialized right there,
+using the parameters of the primary constructor.
+
+If the derived class has no primary constructor, then each secondary constructor has to initialize the base type
+using the `super` keyword, or to delegate to another constructor which does that.
+Note that in this case different secondary constructors can call different constructors of the base type:
+
+```kotlin
+class MyView : View {
+    constructor(ctx: Context) : super(ctx)
+
+    constructor(ctx: Context, attrs: AttributeSet) : super(ctx, attrs)
+}
+```
+
+## Override methods
```

Comment:
Actually, I see nothing wrong when using gerunds as a part of the concept title (when we are not waiting for an actual action from the user) — overriding methods, implementing interfaces, delegating properties, calling Java from Kotlin, etc. 
So maybe we should leave "overriding"?

Reply:
Yep, I was in doubt when changes these titles: does it actually make them better? Let's leave the gerunds if there is no obvious value in changing them to verbs.